Core.Auditing/IfxAuditLogger.cs

```diff
@@ -113,8 +116,29 @@ public static string FetchIPAddress()
                     break;
                 }
             }
-
             return ipAddress;
         }
+        public void LogRequest(ITelemetryClient telemetryClient, TargetResource[] targetResources, CallerIdentity[] callerIdentities, AuditMandatoryProperties auditMandatoryProperties, AuditOptionalProperties auditOptionalProperties = null)
+        {
+            string webAppName = Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME");
+            if (string.IsNullOrEmpty(webAppName))
+            {
+                telemetryClient.LogWarning($"[{nameof(LogRequest)}] Web app name isn't found from environment variable.");
+                webAppName = DefaultWebAppName; // Set to default web app name.
+            }
+            auditMandatoryProperties.AddCallerIdentity(new CallerIdentity(CallerIdentityType.ApplicationID, webAppName));
+            //add audit category
```

Comment:
I don't see the point in comments like these. You are just repeating the name of the function that you are calling.